src/Compilers/CSharp/Portable/Lowering/Instrumentation/DynamicAnalysisInjector.cs

```diff
@@ -129,6 +129,31 @@ private static bool IsExcludedFromCodeCoverage(MethodSymbol method)
             return false;
         }
 
+        private BoundExpression GetFileIndices()
```

Comment:
>private BoundExpression GetFileIndices() [](start = 8, length = 40)

Consider making this a local function because it depends on the proper initialization of _dynamicAnalysisSpans.